CONTRIBUTING.md

```diff
@@ -8,4 +8,8 @@ If your Hacker News implementation meets the [specifications](https://github.com
 
 ## Updating the site
 
-We're exploring using wildcards for our handlers in the `appl.yaml` file but if you add a static file or asset to the repository, you'll have to include it in file in the meantime.
\ No newline at end of file
+We're exploring using wildcards for our handlers in the `appl.yaml` file but if you add a static file or asset to the repository, you'll have to include it in file in the meantime.
+
+## Testing Service Worker locally
+
+To test the Service Worker locally you need to run `npm run precache` before running `bundle exec jekyll serve`.
```

Comment:
Will just need to update this if you decide to change things up in package.json like I mentioned :P